src/main/scala/org/neo4j/spark/service/Neo4jQueryService.scala

```diff
@@ -17,6 +17,12 @@ class Neo4jQueryWriteStrategy(private val saveMode: SaveMode) extends Neo4jQuery
        |${options.query.value}
        |""".stripMargin
 
+  private def createPropsList(props: Map[String, String], prefix: String): String = {
+    props.map(key => {
+      s"${key._2.quote()}: $BATCH_VARIABLE.$prefix.${key._1.removeAlias().quote()}"
+    }).mkString(", ")
+  }
```

Comment:
map to case class for better readibility:
```suggestion
  private def createPropsList(props: Map[String, String], prefix: String): String = {
    props
    .map(key => Neo4jSparkFieldMapping(key._2.quote(), key._1.removeAlias().quote()))
    .map(fieldMapping => {
      s"${fieldMapping.neo4jFieldName}: $BATCH_VARIABLE.$prefix.${fieldMapping.sparkFieldName}"
    }).mkString(", ")
  }
```